### ISSUE TITLE
Replace `ia_collection_s` with multi-value string field `ia_collection`

### ISSUE THREAD

**Issue by cdrini** [internetarchive/openlibrary#6836]

**Comment by hornc** [comment]
@cdrini what are IA collection strings used for in OL?  I haven't looked at this area for some time, but I remember trying to remove this data from OL as it belongs and is best managed by archive.org. The OL stored version could easily get out of sync.

**Comment by cdrini** [comment]
Yeah, the fav- one was annoying :P We removed those. We've gotten requests in the past from partners to view their IA collections in open library. E.g. to view the library explorer with just marygrove books, for example. That field is exposed for those purposes.

**Comment by akramcodez** [comment]
Hi @cdrini @hornc,

I'd like to work on this issue. Should I proceed, or do @hornc's questions need to be resolved first?

Let me know if there's a spec or migration plan I should follow.

**Comment by hornc** [comment]
Thanks @akramcodez for pinging me on this, I forgot that this was still open.

@cdrini I really think `ia_collection` should be removed from OL. This issue lacks a "why" reason, and if it ever was useful, both IA and OL metadata have changed so much it is unlikely to be appropriate. If you think there _is_ a reason, please share and we'll need to come up with a less fragile way of accomplishing whatever feature it used to support.

archive.org collections, the ones that matter, change so this is going to be another syncing nightmare.

I know I have removed `inlibrary` from every record and location in OL's code previously as deprecated.
